python/GafferOFXUI/OFXImageNodeUI.py

```
@@ -0,0 +1,105 @@
+
+		button.clickedSignal().connect( Gaffer.WeakMethod( self.__clicked ) )
+
+	def __clicked( self, button ) :
+
+		with self.context() :
+			if self.getPlug().getValue() :
+				with GafferUI.ErrorDialogue.ErrorHandler(
+					title = "Error loading plugin",
+					parentWindow = self.ancestor( GafferUI.Window )
+				) :
+					with Gaffer.UndoScope( self.scriptNode() ) :
```

Ruff (0.15.15)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

Combine `with` statements

(SIM117)